consensus/src/execution_ctx.rs

```diff
@@ -532,12 +533,17 @@ impl<'a, DB: Database, T: Operations + 'static> ExecutionCtx<'a, DB, T> {
         None
     }
 
-    pub fn get_sortition_config(&self, size: usize) -> sortition::Config {
+    pub fn get_sortition_config(
+        &self,
+        size: usize,
+        exclusion: Option<PublicKeyBytes>,
```

Comment:
@herr-seppia Have you updated the dusk-protocol repo wrt this change in sortition algorithm?

Reply:
I'm currently opening Issues in `dusk-protocol` for all PRs changing the consensus behavior.
I usually do it once the PR is merged, but we might require to do it when the PR is created, so we're sure we don't miss it. 

As for this PR, I'm gonna create the Issue right away